JitsiMeetJS.js

```diff
@@ -147,6 +147,31 @@ var LibJitsiMeet = {
     isDeviceChangeAvailable: function () {
         return RTC.isDeviceChangeAvailable();
     },
+    /**
+     * Returns true if changing the audio output of media elements is supported
+     * and false if not.
+     */
+    isAudioOutputDeviceChangeAvailable: function () {
```

Comment:
I think the names of the methods about input devices are confusing now. I think we should change them to "*InputDevice*" instead of "*Device*" or we should use them without changing the name for the audio output devices too by adding a parameter type ("output"/"input"). 

I think using the existing methods for output devices too is better solution. That way we won't change the existing interfaces. WDYT?